lib/ObjWriter/objwriter.cpp

```diff
@@ -387,14 +389,19 @@ GetSymbolRefExpr(ObjectWriter *OW, const char *SymbolName,
 }
 
 extern "C" void EmitSymbolRef(ObjectWriter *OW, const char *SymbolName,
-                              int Size, bool IsPCRelative, int Delta = 0) {
+                              int Size, bool IsPCRelative, int Delta,
+                              int Kind) {
```

Comment:
I do not think one entrypoint per reloc kind is a good idea. The reloc kind should be enum.

We have a stable reloc kind enum on the JIT-EE interface (it is stable because of it matches COFF reloc kind values where possible). I am going to use that instead.